contributing-guidelines.md

```diff
@@ -0,0 +1,93 @@
+
+## Reviewers and Approvers
+
+As an autoscaler, Karpenter is responsible for minimizing cost, maximizing application runtime, and automatically updating nodes. Its role as a critical cluster component managed by users sets a high bar for contributions and prioritizes efficiency, operationalization, and simplicity for its users. 
+
+At a high level, reviewers and approvers should be inclined to scrutinize the cost-benefit tradeoffs for level of maintenance versus user-benefit, which may materialize as a bias to say “no” in reviews and design discussions. Reviewers should have an initial bias towards coding over reviews to demonstrate a baseline for knowledge of code base. In design discussions, reviewers and approvers should aim to maintain Karpenter’s efficiency, operationalization, and efficiency. 
+
+Karpenter is a customer driven project, building solutions for real customer problems, and delaying solving theoretical ones. Reviewers and approvers should represent these tenets by minimizing changes to Karpenter’s API surface. No API is the best API, as an un-used or dead API becomes a burden for users to reason about, and a further burden for Karpenter users to maintain.
+
+Lastly, as a library vended for and consumed by cloud providers, Karpenter aims to foster participation from all active cloud providers. Karpenter’s set of reviewers and approvers should have representatives from well known cloud provider implementations, as changes to upstream Karpenter can affect all dependent cloud provider implementations.
+
+### Reviewers
+
+Reviewer status is an indication that the person is committed to Karpenter activities, demonstrates technical depth, has accumulated enough context, and is overall trustworthy to help inform approvers and aid contributors by applying a lgtm. Anyone is welcome to review a PR with a comment or feedback even if they do not have rights to apply a lgtm. The requirements listed in the [membership document](https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer) highlight this as well.
+
+The following is a guiding set of criteria for considering a user eligible to be a reviewer:
+
+* Committed - proof of sustained contributions
+* Be a [Kubernetes org member](https://github.com/kubernetes/community/blob/master/community-membership.md#member) (which has its [own set of requirements](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements))
+* Active Karpenter member for at least 6 months
+* Demonstrates technical depth
+* Primary reviewer for at least 5 PRs to the codebase
+* Reviewed or merged at least 10 non-trivial substantial PRs to the codebase
```

Comment:
What is non-trivial defined as? Thoughts on also adding a bullet point for Proposed 3 RFCs?  Reviewers should be actively designing karpenter imo

Reply:
I think RFCs come in tandem with having 10 non-trivial substantial PRs. I think the nature of PRs is that some require an RFC and others don't, especially if those details are figured out elsewhere (like in slack or in working group)

Reply:
Hmm. Fair enough if other methods are enough to hash out a solution then thats fine.